azurerm/resource_arm_virtual_network_gateway_connection_test.go

```diff
@@ -416,6 +439,90 @@ resource "azurerm_virtual_network_gateway_connection" "test_2" {
 `, rInt, rInt2, sharedKey, location, altLocation)
 }
 
+func testAccAzureRMVirtualNetworkGatewayConnection_expressroute(rInt int, location string) string {
+	return fmt.Sprintf(`
+variable "random" {
+  default = "%d"
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-${var.random}"
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctestvn-${var.random}"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  address_space       = ["10.0.0.0/16"]
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "GatewaySubnet"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.test.name}"
+  address_prefix       = "10.0.1.0/24"
+}
+
+resource "azurerm_public_ip" "test" {
+  name                = "acctest-${var.random}"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  allocation_method   = "Dynamic"
+}
+
+resource "azurerm_virtual_network_gateway" "test" {
+  name                = "acctest-${var.random}"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  type     = "ExpressRoute"
+  vpn_type = "RouteBased"
+  sku      = "Basic"
```

Comment:
running the tests it appears this value is unsupported:

> * azurerm_virtual_network_gateway.test: expected sku to be one of [Standard HighPerformance UltraPerformance ErGw1AZ ErGw2AZ ErGw3AZ], got Basic

could we update this (presumably to `Standard`) so that the tests pass? :)